code/datums/martial/boxing.dm

```diff
@@ -222,10 +222,10 @@
 /// Handles our instances of experience gain while boxing. It also applies the exercised status effect.
 /datum/martial_art/boxing/proc/skill_experience_adjustment(mob/living/boxer, experience_value)
 	//Boxing in heavier gravity gives you more experience
-	var/gravity_modifier = boxer.has_gravity() > STANDARD_GRAVITY ? 1 : 0.5
+	var/gravity_modifier = boxer.has_gravity() > STANDARD_GRAVITY ? 1 : 2
 
 	//You gotta sleep before you get any experience!
-	boxer.mind?.adjust_experience(/datum/skill/athletics, round(experience_value * gravity_modifier, 0.1))
+	boxer.mind?.adjust_experience(/datum/skill/athletics, round(experience_value / gravity_modifier, 1))
```

Comment:
This is still going to give less experience at higher gravity, unlike what's stated in the comment.

Reply:
I don't think so?
`gravity_modifier` would be `1` for heavy gravity and `2` for standard and below, and it then divides the experience value by that modifier

Reply:
Oh I read it backwards, thank you.